src/libp2p/connectionsManager.ts

```diff
@@ -58,6 +60,7 @@ export class ConnectionsManager {
   socksProxyAgent: any
   libp2p: null | CustomLibp2p
   localAddress: string | null
+  listenAddrs: string[]
```

Comment:
let's try to be consistent and use either Array or string[] across the whole project, but not both